include/arch/arm/arch/object/smmu.h

```diff
@@ -7,8 +7,13 @@
 #pragma once
 
 
+#if defined(CONFIG_PLAT_ZYNQMP)
+#define SMMU_SID_CNODE_SLOT_BITS    15
+#define SMMU_CB_CNODE_SLOT_BITS     4
+#elif defined(CONFIG_PLAT_TX2)
 #define SMMU_SID_CNODE_SLOT_BITS    8
 #define SMMU_CB_CNODE_SLOT_BITS     6
+#endif
```

Comment:
I wonder if we should move these constants to per-board config so we do not have more #if else when more boards are added.